fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -380,111 +412,211 @@ private enum class Highlight(
 ) {
-    constructor(
-        productRecommendationsEnabled: Boolean? = false,
-        productId: String = "123",
-        reviewGrade: ReviewQualityCheckState.Grade? = ReviewQualityCheckState.Grade.B,
-        analysisStatus: AnalysisStatus = AnalysisStatus.UP_TO_DATE,
-        adjustedRating: Float? = 3.6f,
-        productUrl: String = "",
-        highlights: SortedMap<HighlightType, List<String>>? = sortedMapOf(
-            HighlightType.QUALITY to listOf(
-                "High quality",
-                "Excellent craftsmanship",
-                "Superior materials",
-            ),
-            HighlightType.PRICE to listOf(
-                "Affordable prices",
-                "Great value for money",
-                "Discounted offers",
-            ),
-            HighlightType.SHIPPING to listOf(
-                "Fast and reliable shipping",
-                "Free shipping options",
-                "Express delivery",
-            ),
-            HighlightType.PACKAGING_AND_APPEARANCE to listOf(
-                "Elegant packaging",
-                "Attractive appearance",
-                "Beautiful design",
-            ),
-            HighlightType.COMPETITIVENESS to listOf(
-                "Competitive pricing",
-                "Strong market presence",
-                "Unbeatable deals",
-            ),
-        ),
-        recommendedProductState: ReviewQualityCheckState.RecommendedProductState =
-            ReviewQualityCheckState.RecommendedProductState.Initial,
-        productVendor: ReviewQualityCheckState.ProductVendor = ReviewQualityCheckState.ProductVendor.AMAZON,
-    ) : this(
-        productRecommendationsEnabled = productRecommendationsEnabled,
-        productAnalysis = AnalysisPresent(
-            productId = productId,
-            reviewGrade = reviewGrade,
-            analysisStatus = analysisStatus,
-            adjustedRating = adjustedRating,
-            productUrl = productUrl,
-            highlights = highlights,
-            recommendedProductState = recommendedProductState,
-        ),
-        productVendor = productVendor,
-    )
-}
+    Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
+        ReviewQualityCheckCard(
+            modifier = Modifier
+                .fillMaxWidth()
+                .clickable(onClick = onRecommendationClick),
+        ) {
+            Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
+                Text(
+                    text = stringResource(R.string.review_quality_check_ad_title),
+                    color = FirefoxTheme.colors.textPrimary,
+                    style = FirefoxTheme.typography.headline8,
+                )
+
+                Row(horizontalArrangement = Arrangement.spacedBy(12.dp)) {
+                    val imageSize = with(LocalDensity.current) { 60.dp.toPx() }.roundToInt()
+                    AsyncImage(
+                        model = ImageRequest.Builder(LocalContext.current)
```

Comment:
Could We just set the url here, it would use the singleton `imageLoader` ?

Reply:
Yeah, that's an option too! My initial thinking was we could use image request builder so we set the pixel dimensions of the image, so we're not necessarily holding onto the full image size in memory.

Reply:
Gotcha, makes sense! Wondering if the modifier size param already doesn't indicate that to `AsyncImage`?